### ISSUE TITLE
[TEST] Sporadic Failure, nodeos_voting_lr_test

### ISSUE THREAD

**Issue by lazaridiscom** [EOSIO/eos#5036]
On ubuntu 18.04 64bit, VirtualBox on Win10pro, quite slow i7 laptop

Using 1.1.3 (latest master)

## Problem

All tests pass, except `nodeos_voting_lr_test`, which fails sporadically 

Launching test alone via

```bash
ctest --output-on-failure -R nodeos_voting_lr_test > nodeos_voting_lr_test6.log 2>&1
```
resulted in 2 failures out of 5 runs.

Attaching the log obtained from redirecting the output, zipped, due to size

[nodeos_voting_lr_test6.zip](https://github.com/EOSIO/eos/files/2259285/nodeos_voting_lr_test6.zip)

**Comment by brianjohnson5972** [comment]
This should be fixed by #4972 when it is merged.

**Comment by lazaridiscom** [comment]
@brianjohnson5972 , sounds good.

As for this Issue: Could you please reopen the issue? It should be closed after the fix is in production/verified.
